content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -196,6 +196,108 @@ add a brief description in the :guilabel:`Description` field. Click :guilabel:`F
 the :guilabel:`Rule Configuration` wizard. Now, the new rule is active and devices can connect to
 the Windows virtual :abbr:`IoT (Internet of Things)` box.
 
+Worldline exception
+~~~~~~~~~~~~~~~~~~~
+
+Worldline is a payment terminal that can be connected to Odoo's point of sale system. It allows for
+a comprehensive and fluid payment experience for customers. Worldline is available in Benelux
+(coalition of Netherlands, Belgium, Luxembourg). When using the Windows IoT server to connect the
+Worldline payment terminal it is necessary to create an exception in the Windows firewall so that
+a connection can be made between the Odoo database/:abbr:`IoT (Internet of Things)` box and
+Worldline.
```

Comment:
```suggestion
Worldline is a payment terminal that can be connected to Odoo's point of sale system. It allows for
a comprehensive and fluid payment experience for customers. Worldline is available in Benelux
(coalition of Belgium, the Netherlands, and Luxembourg).

When using the Windows IoT server to connect the Worldline payment terminal, it is necessary to create
an exception in the Windows firewall so that a connection can be made between the Odoo
database/:abbr:`IoT (Internet of Things)` box and Worldline.
```

Reply:
Just broke up the paragraph and reworded some stuff for clarity.

A suggestion for the first sentence, though: if the sentence "...connected to Odoo's point of sale system" is referring to the PoS app, I would word it like "...connected to Odoo's *Point of Sale* app" ... what do you think?